app/src/main/res/values/donottranslate.xml

```diff
@@ -72,6 +74,7 @@
     <string name="subtype_generic_dvorak">%s (Dvorak)</string>
     <string name="subtype_generic_colemak">%s (Colemak)</string>
     <string name="subtype_generic_workman">%s (Workman)</string>
+    <string name="subtype_generic_trf">%s (F layout)</string>
```

Comment:
This is not correct. `(F layout)` layout will appear for all `QWERTY` based languages like English, will allow making a custom layout, and then show Turkish letters on it.

This has to be limited to Turkish language. One way to do it would be like `subtype_in_root_locale_hi_ZZ` above. It would be shown as an extra language.